### DISCUSSION TITLE
Can't uninstall

### DISCUSSION THREAD

**Discussion by Poomurugan571** [blueCFD/Core#232] [Q&A] [answered]
Hi,
I tried to install bluecfd in my pc,
while installing it gave me following warning (attached the screenshot).

now, i am trying to uninstall but its not uninstalling  again

 giving following warning (attached the screenshot).

can anyone please help me to solve the issue.
thank you,

![Capture](https://user-images.githubusercontent.com/49983919/56736469-d8f8b500-6785-11e9-80fb-47d851200837.PNG)
![Capture_01](https://user-images.githubusercontent.com/49983919/56736470-d8f8b500-6785-11e9-8f75-69931e5ac291.PNG)

**Comment by wyldckat** [comment, marked as answer]
With Windows Explorer (it's for exploring files and folders, not Internet Explorer), go into this folder: `C:\Users\Murugan\AppData\Roaming\blueCFD-Core-2017\Start Menu\Installation`

Then double-click on the uninstall icon. That should allow you to uninstall it with success.

As for the second image, I've finally understood that it appeared when installing it. You can ignore that error message.

In the meantime, I noticed that you installed blueCFD-Core 2017-1. Instead of that version, please install blueCFD-Core 2017-2, which has several bugs that were fixed after the release of 2017-1.
